Wrappers.h

```diff
@@ -0,0 +1,55 @@
+#include "Safeway.h"
+#include "ShelterOfCats.h"
+#include "Artist.h"
+#include "CommerceBureau.h"
+#include "Map.h"
```

Comment:
Don't include what you don't use.
You aren't using any of these types in this header file.
so Forward Declare them.
You **ARE** using them in the cpp file, which is where these header files should be included.

- Be aware that you **cannot** forward declare nested types.
- You are missing a `#pragma once`
- [ ] revise